CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(ed_robocup)
 
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++11)
```

Comment:
Isn't using `add_definitions` better? It is the standard way of adding pre-processor directives

Reply:
Nope, that should be used in a catkin packages. As this would result that this is applied to all following packages. This is not the case with add_compile_options. This is only applied to this package.